arch/arm64/boot/dts/freescale/crocodile.dtsi

```diff
@@ -282,6 +282,38 @@
 	};
 };
 
+&ecspi3 {
+	fsl,spi-num-chipselects = <6>;
+	cs-gpios = <0>, <&gpio5 1 0>, <&gpio4 25 0>, <&gpio4 26 0>, <&gpio4 23 0>, <&gpio5 3 0>;
```

Comment:
Something like
```
cs-gpios = <0>,
           <&gpio5  1 GPIO_ACTIVE_HIGH>,
           ...
           <&gpio5  3 GPIO_ACTIVE_HIGH>;
```
would make it easier to read.

Reply:
:+1: